compiler/control/OMROptions.cpp

```diff
@@ -1793,6 +1793,9 @@ OMR::Options::Options(
    {
    TR_ASSERT(optimizationPlan, "Must have an optimization plan when calling this method");
 
+   //initialize all options to false (0)
+   memset(_options, 0, sizeof(_options));
```

Comment:
Do we need this line? On line 1810 we copy every field from "this" with fields from "mainOptions". I assume that the embedded array `_options` is going to be copied over as well.

Reply:
Regarding _jitCmdLineOptions, do we not need to re-initialize it, as was done previously by using memset to set everything to 0?

Reply:
The role of the memset has been superseded by the new constructor which initializes all fields.
When `_jitCmdLineOptions = new (PERSISTENT_NEW) TR::Options();` is executed, I expect all fields to be set to 0.